addons/account/tests/test_reconciliation_matching_rules.py

```diff
@@ -985,3 +986,105 @@ def create_payment_line(amount, memo, partner):
         self._check_statement_matching(self.rule_1, {
             self.bank_line_1.id: {'aml_ids': (pmt_line_1 + pmt_line_2).ids, 'model': self.rule_1, 'partner': payment_partner},
         }, statements=self.bank_line_1.statement_id)
+
+    def test_ceci_est_un(self):
+        country = self.env.ref('base.us')
+        tax_report = self.env['account.tax.report'].create({
+            'name': "Tax report",
+            'country_id': country.id,
+        })
+        tax_report_line = self.env['account.tax.report.line'].create({
+            'name': 'test_tax_report_line',
+            'tag_name': 'test_tax_report_line',
+            'report_id': tax_report.id,
+            'sequence': 10,
+        })
+        tax_tag_pos = tax_report_line.tag_ids.filtered(lambda x: not x.tax_negate)
+        tax_tag_neg = tax_report_line.tag_ids.filtered(lambda x: x.tax_negate)
+        tax = self.env['account.tax'].create({
+            'name': 'GST Inc Sale (10%)',
+            'type_tax_use': 'sale',
+            'amount': 10,
+            'amount_type': 'percent',
+            'price_include': True,
+            'invoice_repartition_line_ids': [
```

Comment:
remove `price_include`, `amount_type`, `type_tax_use` and rename the tax to something like `test tax`